### PR TITLE
Settings button

### PR DESCRIPTION
**Pull request by pari-27**
Settings icon button created

### CODE REVIEW

File: assets/js/buttons-examples.js [hunk 1]
```diff
@@ -216,6 +216,13 @@ $(document).ready(function () {
           ],
         },
 
+        {
+          type: "setting",
+          label: "Setting Button",
+          classes: "setting-btn",
+          text: "",
+        },
+
```
Comment (HADES-01) [inline assets/js/buttons-examples.js:225]: Arrange it according to alphabetical order.

Comment (HADES-01) [inline assets/js/buttons-examples.js:225]: Please arrange it in alphabetical order.

**Review by HADES-01 — CHANGES_REQUESTED** [review]
@pari-27 Great work, Just do the mentioned changes

File: assets/js/buttons-examples.js (hunk 1 above)
Comment (pari-27) [reply assets/js/buttons-examples.js]: Done!!

File: assets/js/buttons-examples.js [hunk 2]
```diff
@@ -278,6 +277,12 @@ $(document).ready(function () {
           classes: "scroll-to-top-btn",
           text: "",
         },
+        {
+          type: "setting",
+          label: "Setting Button",
```
Comment (shahednasser) [inline assets/js/buttons-examples.js:282]: Please also remove "Button" from the label

**Review by shahednasser — CHANGES_REQUESTED** [review]

**Comment by shahednasser** [comment]
Great work thank you!

@all-contributors please add @pari-27 for code

**Pull request merged**